### PR TITLE
Fix footer and absolute-center

### PR DESCRIPTION
**Pull request by rubene**
🎨 🐘 
* Add a wbenkid-transforms because transform is not supported by old
safari. causing the button-grouping content to render outside the div.
* Add heights vars to add as margin and height variables to the footer
because flex-box does not work on IE or safari.

### CODE REVIEW

**Comment by jessedoyle** [comment]
PR is also submitted to Foundation core to fix the `absolute-center` mixin: https://github.com/zurb/foundation-sites/pull/9077.

File: app/assets/stylesheets/ama_layout/layout_components/footer.scss [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+  }
+  @include breakpoint(medium only){
+    margin-bottom: -$footer-push-medium;
+  }
+  @include breakpoint(large up){
+    margin-bottom: -$footer-push-large;
+  }
+
+  &:after{
+    content: "";
```
Comment (jbabe) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: Can we drop this line? It looks like it's not adding anything?

**Comment by jbabe** [comment]
💩

File: app/assets/stylesheets/ama_layout/layout_components/footer.scss (hunk 1 above)
Comment (jessedoyle) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: We adapted the code from here: https://css-tricks.com/snippets/css/sticky-footer/.

If I remove the `content` property it breaks the footer, so I assume it's needed.

**Comment by jessedoyle** [comment]
:wave: - There are a few markup changes that we need to push now to make use of this footer fix.

**Comment by jbabe** [comment]
🐘 🐘 🐘

**Comment by jbabe** [comment]
:wave: :art:

**Comment by jbabe** [comment]
Or do we need to wait for @kaytcampbell's PR to go in before merging this?

**Comment by jessedoyle** [comment]
Maybe we should wait for @kaytcampbell's PR (#139).

File: app/assets/stylesheets/ama_layout/layout_components/footer.scss [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
-// See: https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/
-.off-canvas-wrapper{
-  display: flex;
-  flex-direction: column;
-  min-height: 100vh;
+$footer-push-small: 617px;
+$footer-push-medium: 432px;
+$footer-push-large: 329px;
+
+html, body{
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: Once Kayt's PR is merged you shouldnt need this css anymore.

**Comment by jessedoyle** [comment]
:wave:

File: app/assets/stylesheets/ama_layout/layout_components/footer.scss (hunk 2 above)
Comment (jessedoyle) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: Are the `html` and `body` tags getting height 100% elsewhere then?

Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: Can you try removing this CSS? If you still need it I dont think it should live in a footer file.

Comment (jessedoyle) [inline app/assets/stylesheets/ama_layout/layout_components/footer.scss]: Yea it looks like the css can be removed without issue.

**Comment by jessedoyle** [comment]
:wave:

File: app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss [hunk 3]
```diff
@@ -61,6 +61,7 @@
 
   &__content{
     @include absolute-center;
+    -webkit-transform: translateX(-50%) translateY(-50%);
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss]: Not sure where absolute-center comes from but if the only goal here is the vertically align then foundation has an include that does the transform as well: vertical-center

Comment (jessedoyle) [inline app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss]: The `absolute-center` mixin comes from Foundation.

There's actually an issue with the code in Foundation core: https://github.com/zurb/foundation-sites/pull/9077

Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss]: Okay, sounds good.

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Pull request merged**